wordpress-mobile/WordPress-Android
# Issues/8560 analytics tweaks

Fixes #8560 

This PR adds some events to WPAndroid that already exist in WPiOS, and adds/restores a `dotcom_user` user info property to all events. 

To test:
Apply the patch.
Open plans, Sharing management, and Sharing button management screens. 
Check tracks live events, filter by your user name.
Confirm that site_menu_opened events appear with a menu_type property for each screen.
Confirm that there is a user_info_dotcom_user event property present. 

For now it *looks* like account_created is what we want with the addition of the dotcom_user property.

@theck13 could I trouble you for a review of this one?
